src/hooks/useLoadUpdateDealDeps.ts

```diff
@@ -138,8 +138,10 @@ const useLoadUpdateDealDeps: UseLoadUpdateDealDeps = (dealId) => {
             priorities,
             accountInfo,
         ].every(({ isLoading }) => Boolean(isLoading)),
-        deal: get(deal, ["data", "properties"], {}) || {},
+        deal: get(deal, ["data", "properties"]) as Deal["properties"],
```

Comment:
We don't need lodash here, native JS/TS has us covered with optional chaining. Also we shouldn't be typehinting here, it should be up near `const deal = useQueryWithClient` if that is the type it is - just so TS is more aware of what it is for longer.

Reply:
here story to refactoring with `lodash` https://app.shortcut.com/deskpro/story/90992/hubspot-improvements